src/components/Allproducts.js

```diff
@@ -20,15 +22,20 @@ export default function Allproducts() {
   });
 
   return (
-    <div>
+    <div className='allproducts'>
```

Comment:
Let's put a dash in allproducts for readability.

Reply:
Will do so on the second pass.